src/Form/Form.js

```diff
@@ -49,22 +48,24 @@ class Form {
         }
 
         this.getValues = () => {
-            const credentials = [...this.inputs.credentials, ...this.inputs.emailNew].reduce((output, input) => {
+            const credentials = [...this.inputs.credentials, ...this.inputs.identities].reduce((output, input) => {
                 const subtype = getInputSubtype(input)
-                output[subtype] = input.value || output[subtype]
+                if (['username', 'password', 'emailAddress'].includes(subtype)) {
+                    output[subtype] = input.value || output[subtype]
+                }
                 return output
             }, {username: '', password: ''})
             // If we don't have a username, let's try and save the email if available.
-            if (credentials.emailNew && !credentials.username) {
-                credentials.username = credentials.emailNew
+            if (credentials.emailAddress && !credentials.username) {
+                credentials.username = credentials.emailAddress
             }
-            delete credentials.emailNew
+            delete credentials.emailAddress
             return credentials
         }
 
         this.hasValues = () => {
-            const {username, password} = this.getValues()
-            return !!(username && password)
+            const {password} = this.getValues()
+            return !!password
```

Comment:
This was incorrect. We want to return `true` here even if only the password is present.